code/style.css

```diff
@@ -145,6 +184,31 @@ button {
   color: var(--primary);
 }
 
+/* START WINDOW */
+.start-wrapper {
+  display: none;
+  align-items: center;
+  justify-content: center;
+  background: var(--secondary);
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  z-index:1;
+  height: 100vh;
+}
+
+.start-div {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  max-width: 700px;
+  text-align: center;
+}
+
+
```

Comment:
Dayra You did the game like I wish I had done mine SO IMPRESSIVE!!! I love the confetti, starter window, and custom popup!!!!! the only thing I think you could think about is to name the variables/tags to more describing - like starter window (if not the modal window is the right description for this variable and I just don't know that :) ) Good work!!! - -[ YES]  Am I able to understand the code easily?
- [NO ]  Does the code have lots of duplicates? Could it be tidied up?
- [ NO]  Does it work or does anything look broken?
- [YES]  Does it follow the general and blue requirements?

Reply:
Thank you Jessica for your code review! :) Really great inputs that I will apply for my next projects :)